utils/after_success.sh

```diff
@@ -0,0 +1,24 @@
+#!/bin/bash
+
+# This script is meant to be used by Travis-CI to publish artifacts (binary, source and javadoc jars) when releasing snapshots.
+# Right now it will be manually triggered to push jars to nexus server.
+
+SITE_VERSION=$(./gradlew properties | grep -o 'version.*' | sed -n -e 's/^.*version: //p')
```

Comment:
What occurs before `version` on the output of `./gradlew properties`? If it is just whitespace, then use `\s*` instead of `.*`. The `.*` could be too grabby if there are new lines in the future that contain `version`. Add all these things to the first grep pattern too. (If you match just 'version' in the first one, there are cases where the second replacement won't happen.)